bin/setup

```diff
@@ -6,25 +6,17 @@ if ARGV.any?
 Usage: bin/setup
 
 Environment Variable Options:
-    SKIP_DATABASE_SETUP  Skip the creation, migration, and seeding of the
-                         database.
+    SKIP_DATABASE_SETUP  Skip the creation, migration, and seeding of the database.
+    SKIP_UI_UPDATE       Skip the update of UI assets.
     SKIP_TEST_RESET      Skip the creation of the test enviroment.  Defaults to
                          true in production mode since the tasks do not exist.
 EOS
   exit 1
 end
 
-def skip_ui_update?
-  ENV["CI"]
-end
-
-def skip_database_setup?
-  ENV["SKIP_DATABASE_SETUP"] || ENV["CI"]
-end
-
-def skip_test_reset?
-  ENV["SKIP_TEST_RESET"]
-end
+ENV["SKIP_DATABASE_SETUP"] = "true" if ENV["CI"]
+ENV["SKIP_UI_UPDATE"] = "true" if ENV["CI"]
+ENV["SKIP_TEST_RESET"] = "true" if ENV['RAILS_ENV'] == 'production'
```

Comment:
This bit gets rid of these local methods in favor of matching the way it's done in bin/update.